### ISSUE TITLE
HTML5 Markup cleaned up

### ISSUE THREAD

**Issue by wuuti** [splittingred/TinyMCE#92]
It seems that the current Tiny version of the Modx package (4.3.3-pl) does not support html5-tags. (see http://forums.modx.com/thread/?thread=78709&page=1). 
Whenever you try to add tags like figure/figcaption/article they get removed or replaced with p by the cleanup routines. 

Configuration of xconfig.js (tinymce_extended_valid_elements) did not work.
Anyway, this configuration should be moved into Modx system settings, just as "invalid_elements" is available there.

**Comment by rosstimson** [comment]
I'm eager for a fix here too as it causes me major headaches with a couple of existing sites.  I checked the xconfig.js as mentioned in several threads and the HTML5 elements are already listed there, however it does not work.

I noticed the official TinyMCE site shows a HTML5 demo here:  http://www.tinymce.com/tryit/html5_formats.php

Maybe the version of TinyMCE included in the MODX plugin needs to be bumped, although I'm not sure how much work this would be and the latest 4.0 version still looks to be in beta.

**Comment by photoman** [comment]
Yes.. PLEASE fix this..  HTML5 elements should be allowed - this feels like the IE 6 days...when stuff should be working... ugh. OR can someone provide docs how to install the latest TinyMCE so that it works with MODX.

In the short term, look for  CKEditor in the MODX Package Manager, seems to like  HTML5 elements. :)

**Comment by pdincubus** [comment]
The example demo page is using TinyMCE 4.0.6, the current MODX extra version is an older 3.x version. I've forked this to try and start integrating the new version but seem to only be able to break it as the directories and filenames are all different in the major release bump!

**Comment by pdincubus** [comment]
Also worth noting that Redactor editor is available from ModMore which is HTML5 out of the box, but you'll have to pay for it!
